qualang_tools/results/results.py

```diff
@@ -36,15 +36,27 @@ def __init__(self, job, data_list, mode="wait_for_all"):
                     self.data_handles[-1].wait_for_values(1)
                 else:
                     raise Warning(f"{data} is not saved in the stream processing.")
+            # Live plotting parameters
+            self._b_cont = False
+            self._b_last = True
+            self.start_time = 0
 
     def is_processing(self):
         """
         Returns True while the program is processing. Used for live plotting.

```

Comment:
"Returns True while the program is processing, and also once after the processing is done. Can be used for live plotting."

Reply:
done